### ISSUE TITLE
Clicking "Give me everything..." from /gaming displays middle of page

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#793]
```
Click "give me everything..." from /gaming and you're linked to 
http://master.html5rocks.appspot.com/en/tutorials#gaming which scrolls the page 
to the bottom.

I would expect this to display the top of the page.

See screenshot.
```

Original issue reported on code.google.com by `sethladd@google.com` on 1 Feb 2012 at 2:54

Attachments:
- [Screen shot 2012-01-31 at 6.52.45 PM.png](https://storage.googleapis.com/google-code-attachments/html5rocks/issue-793/comment-0/Screen shot 2012-01-31 at 6.52.45 PM.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 1 Feb 2012 at 4:12
- Changed state: **Assigned**
- Added labels: **Priority-P1**, **Milestone-Q12012**, **Updates**
- Removed labels: **Priority-P2**

**Comment by GoogleCodeExporter** [comment]
```
Hanrui, the checboxes add 'audience:' prefix:
/tutorials#audience:business

Whereas, you  click a 'business'  tag next to an article and get:
/tutorials#business

Can we make sure both are the same?
```

Original comment by `ericbide...@html5rocks.com` on 1 Feb 2012 at 4:17

**Comment by GoogleCodeExporter** [comment]
```
A srollTo top might be necessary after a tag is clicked.

BTW, Fixed the persona pages to point to /tutorials/#audience:x. But the 
behavior from
my last comment is still there.
```

Original comment by `ericbide...@html5rocks.com` on 1 Feb 2012 at 4:53

**Comment by GoogleCodeExporter** [comment]
```
"/tutorials#audience:business" and "/tutorials#business" are not exactly same 
thing.
the former means that you get here by clicking business checkbox in audience 
list, so in the result page, the business checkbox will also get checked for 
your convenience.

the latter means "giving me all things that has business tag" so you won't see 
the checkbox get checked in audience list in the result page.

But they both check the tags of each article, so they will return you the same 
articles. The only difference is whether you will get the checkbox checked.

```

Original comment by `han...@google.com` on 6 Feb 2012 at 2:07
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Why wouldn't they be treated exactly the same?
```

Original comment by `ericbide...@html5rocks.com` on 6 Feb 2012 at 5:33

**Comment by GoogleCodeExporter** [comment]
```
Because:
1) not all tags (which are displayed after article titles) are options in 
audience list
2) we should opt-in/opt-out the options in respect to users' behaviours, if the 
user clicks the options, that means he's aware of the checkboxes, so we display 
the result page with the checkbox checked. But if the user just click one tag 
to get to the result page, we should not check the checkbox cause he may not 
aware of these checkboxes.
```

Original comment by `han...@google.com` on 7 Feb 2012 at 2:33

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision c771766b2754.
```

Original comment by `han...@google.com` on 9 Feb 2012 at 3:01
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
FYI, I opened another issue 
here(http://code.google.com/p/html5rocks/issues/detail?id=806) based on this 
one.
```

Original comment by `han...@google.com` on 9 Feb 2012 at 3:31
